CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ## 💡 Enhancements 💡
 
 - Remove expand cases that cannot happen with config.Map (#4649)
+- Add `max_request_body_size` to otlpreceiver's http settings (#4677)
```

Comment:
This is not correct, this adds the option to all the receivers that use the HttpClientSettings and not just otlp, consider to update everywhere

Reply:
CHANGELOG updated